File: Bazar/Bazar.js
function main() {
    var elephant = 10;
    var bicycles = 10;
    var sunflower = 10;
    const initialPrice = elephant + bicycles + sunflower;
    elephant += 3;
    sunflower -= 5;
    bicycles = elephant + 2;
    const finalPrice = elephant + bicycles + sunflower;

    const gainOrLoss = finalPrice - initialPrice;

    const prefix = gainOrLoss > 0 ? 'Gained' : 'Lost';
    const result = `${prefix} ${Math.abs(gainOrLoss)}$`;

    console.log(result);
}

main();

File: Bazar/Bazar.java
public class Bazar {

    public static void main(String[] args) {
        int elephant = 10;
        int sunflower = 10;
        int bicycle = 10;
        int initialPrice = elephant + sunflower + bicycle;

        elephant += 3;
        sunflower -= 5;
        bicycle = elephant + 2;
        int finalPrice = elephant + sunflower + bicycle;

        int gainOrLoss = finalPrice - initialPrice;
        String result = gainOrLoss > 0 ? "gained " : "lost ";

        System.out.println("In the end, the bazar seller has " + result + Math.abs(gainOrLoss) + "$");
    }
}
